Frontend/src/components/pages/partial/SearchBar.jsx

```diff
@@ -25,38 +29,40 @@ export default function SearchBar({ type, data, setSearchResults }) {
     setSearchResults(results);
   }
 
-  return (
-    <Autocomplete
-      freeSolo
-      size="small"
-      onInputChange={(e, inputValue) => {
-        handleChange(inputValue);
-      }}
-      options={data.map((item) => ({
-        value: item.id,
-        label:
-          type === "users" ? `${item.firstName} ${item.lastName}` : item.name, // TODO: collaborations label?
-      }))}
-      renderOption={(props, option) => (
-        <li {...props} key={option.value}>
-          {option.label}
-        </li>
-      )}
-      renderInput={(params) => (
-        <TextField
-          {...params}
-          label={`Search ${type}`}
-          InputProps={{
-            ...params.InputProps,
-            startAdornment: (
-              <InputAdornment position="end">
-                <SearchIcon />
-              </InputAdornment>
-            ),
+  if (data !== undefined && data !== null && data.length !== 0) {
```

Comment:
isto pitanje vrijedi i za ovo: jel postoji situacija u kojoj ce nesto od ovog biti false

Reply:
Ispravak: ovo vratiti kako je bilo (bez if-a) jer krši layout, zelim da bude search bar i kada name nicega u tablici

Reply:
Da, i taj if to omogućuje, da kada nema ničega da se prikaže ništa.

Reply:
Komentar na watsappu sa slikom.